witchcraft/server_routes.go

```diff
@@ -48,6 +49,11 @@ func (s *Server) addRoutes(mgmtRouterWithContextPath wrouter.Router, runtimeCfg
 	if err := addDebuggingRoutes(mgmtRouterWithContextPath); err != nil {
```

Comment:
nit: we may want to change this to `addPprofRoutes()` or similar to disambiguate the debug routes below.

Also need to update the comment to make it plural `// add debugging endpoints ...`